packages/twenty-server/src/workspace/workspace-sync-metadata/services/workspace-sync-field-metadata.service.ts

```diff
@@ -63,6 +63,10 @@ export class WorkspaceSyncFieldMetadataService {
 
     // Loop over all standard objects and compare them with the objects in DB
     for (const customObjectMetadata of customObjectMetadataCollection) {
+      if (!customObjectMetadata.standardId) {
```

Comment:
Why are we checking standardId on a "custom" ObjectMetadata? It should always be false or am I missing something?

Reply:
You're totally right, it's actually the bug @charlesBochet got with custom object !